addons/website_slides_survey/models/slide_channel.py

```diff
@@ -2,9 +2,30 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from odoo import fields, models
+from odoo.osv import expression
 
 
 class Channel(models.Model):
     _inherit = 'slide.channel'
 
     nbr_certification = fields.Integer("Number of Certifications", compute='_compute_slides_statistics', store=True)
+
+    def _remove_membership(self, partner_ids):
+        """Clear certifications' user_inputs invite_token.
+        
+        Deleting the user_inputs' invite_token allows to keep track of the current
+        pool of attempts allowed since the user (last) joined the course, as only 
+        those will have an invite token, that will  be reused until there are no 
```

Comment:
As you'll fix for the runbot, there is also this one ;)
```suggestion
        those will have an invite token, that will be reused until there are no 
```